src/components/Method/Method.js

```diff
@@ -6,20 +6,21 @@ import BodyContent from '../BodyContent/BodyContent'
 import Description from '../Description/Description'
 import Parameters from '../Parameters/Parameters'
 import Response from '../Response/Response'
+import MethodPath from './MethodPath'
 import { styles } from './Method.styles'
 
 @styles
 export default class Method extends PureComponent {
   render () {
-    const { method, classes, initialSchemaTreeDepth } = this.props
-    const { title, type, description, parameters, request, responses } = method
+    const {method, classes, initialSchemaTreeDepth} = this.props
+    const {title, type, path, paths, description, parameters, request, responses} = method
 
     return (
       <ScrollableAnchor id={method.link}>
         <div className={classes.method}>
           <h3>
             {title}
-            <span className={classNames(classes.type, `${method.type}`)}>{type}</span>
+            <MethodPath type={type} path={path} paths={paths} classes={classes}/>
```

Comment:
No need to push classes down. Any relevant classes should be moved to the MethodPath component instead.